packages/sit-onyx/src/components/OnyxBadge/OnyxBadge.vue

```diff
@@ -0,0 +1,91 @@
+<script lang="ts" setup>
+import { useDensity } from "../../composables/density";
+import type { OnyxBadgeProps } from "./types";
+
+const props = withDefaults(defineProps<OnyxBadgeProps>(), {
+  variation: "primary",
+});
+
+const { densityClass } = useDensity(props);
+
+defineSlots<{
+  /**
+   * Badge content.
+   */
+  default(props: Record<string, never>): unknown;
```

Comment:
```suggestion
  default(): unknown;
```

This dummy type is no longer needed since the latest Storybook update :)

Reply:
Yes I saw that in your PR, but when I removed them the slots were not available. I'll try again

Reply:
You probably have to merge the latest changes from main into your branch :)

Reply:
When I remove the type, nothing is shown
![image](https://github.com/SchwarzIT/onyx/assets/60856270/b977a4cd-a641-4cc1-be60-139bdd37555b)

Reply:
okay, now everything is red and I cannot run the project

![image](https://github.com/SchwarzIT/onyx/assets/60856270/19c8aa74-5f19-4445-99c7-b2de58edd79d)

Reply:
Whoah 😮 That looks like a Artifactory issue. Does running "pnpm login" solve this?